pages/locations/[slug]/reviews.vue

```
@@ -208,19 +208,10 @@ if (!siteId) throw createError({ statusCode: 404 })
-  { key: () => `loc-reviews-loc-${siteId}-${slug.value}`, default: () => ({ location: null }) }
-)
-const location = computed(() => (locData as ApiValue).value?.location ?? null)
-
-const { data, pending } = await useFetch(
-  () => `/api/public/sites/${siteId}/locations/${slug.value}/reviews`,
-  { key: () => `loc-reviews-${siteId}-${slug.value}`, default: () => ({ aggregate: null, reviews: [] }) }
-)
-
-const aggregate = computed(() => (data as ApiValue).value?.aggregate ?? null)
-const reviews = computed(() => (data as ApiValue).value?.reviews ?? [])
+const { location, locationReviews, reviewsAggregate, reviewsList, data: bootstrapData } = useBootstrap()
```

ESLint

[error] 'locationReviews' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)